core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/DataAccess.scala

```diff
@@ -32,11 +34,21 @@ trait DataAccess
 
   import driver.api._
 
+  // only called from TestDriverComponent
   def truncateAll: WriteAction[Int] = {
     // important to keep the right order for referential integrity !
     // if table X has a Foreign Key to table Y, delete table X first
 
-    TableQuery[EntityAttributeTable].delete andThen             // FK to entity
+    // davidan: instead of specific ordering, could we instead SET FOREIGN_KEY_CHECKS = 0; truncate tables ...; SET FOREIGN_KEY_CHECKS = 1; ?
```

Comment:
I have a feeling the reason that we don't already do this is because we didn't think to do it before. Agreed that it might be nice to simplify this function at some point later on